### DISCUSSION TITLE
minio pool offline a node

### DISCUSSION THREAD

**Discussion by zxl716** [minio/minio#16949] [Q&A] [answered]
Minio has 2 pools, one of which consists of 8 nodes, and each node has 8 hard disks, 1T. If a node goes offline, what is the specific operation? At present, the cluster space is sufficient, and I just want to take the nodes in the pool offline and recycle them. Thanks

**Comment by harshavardhana** [comment, marked as answer]
You can lose up to your parity number of servers for both pools. Pools are only meant for expansion they cannot be independently entirely taken down. 

You need quorum on both pools to be operational.

**Reply by zxl716** [reply]
thank you for your reply. If there is only one pool, a node in the pool is recycled. Any good suggestions?
